packages-web/maps/src/components/GoogleMap.ts

```diff
@@ -13,9 +13,15 @@ type DataSourceLocationProps = Container.DataSourceLocationProps;
 type Location = Container.Location;
 type SharedProps = MapUtils.SharedProps;
 
+export interface GoogleMapsClickEvent extends google.maps.MouseEvent {
+    options: {
+        GUID?: string;
```

Comment:
Naming convention to use only single Capital `Guid` instead of `GUID`

Reply:
Fixed